lib/Models/WebProcessingServiceCatalogFunction.js

```diff
@@ -478,34 +479,38 @@ WebProcessingServiceCatalogFunction.prototype.invoke = function() {
         content: inputsSection
     });
 
-    var uri = new URI(this.url).query({
-        service: 'WPS',
-        request: 'Execute',
-        version: '1.0.0',
-        Identifier: this.identifier,
-        DataInputs: createWpsDataInputsFromParameters(this)
-    });
+    var that = this;
+    when.all(createWpsDataInputsFromParameters(this)).then(function(dataInputs) {
```

Comment:
It's in Geoglam's GeoJsonParameter, which is being added to WebProcessingServiceCatalogFunction's parameterConverters in the index file of Geoglam.